flows/exam-1.yml

```diff
@@ -28,24 +28,28 @@ rules:
         permissions: []
 
     -
-        if_in_facility: test_center
-        if_session_duration_shorter_than_minutes: 0.5
-        permissions: [view, submit_answer, end_session, see_session_time, lock_down_as_exam_session]
+        if_has_role: [instructor]
+        permissions: [view, submit_answer, change_answer, end_session, see_correctness, see_session_time]
 
     -
-        if_in_facility: test_center
-        message: "You exam will end soon."
-        if_session_duration_shorter_than_minutes: 1
-        permissions: [view, submit_answer, end_session, see_session_time, lock_down_as_exam_session]
+        if_in_facility: "cbtf"
+        if_session_duration_shorter_than_minutes: 45  # length until warning appears
+        permissions: [view, submit_answer, change_answer, end_session, see_correctness, see_session_time, lock_down_as_exam_session]
 
     -
-        if_in_facility: test_center
+        if_in_facility: "cbtf"
+        message: "Your exam will end soon."
+        if_session_duration_shorter_than_minutes: 120  # keep in mind DRES students!
```

Comment:
- DRES is UIUC-specific terminology (should use neutral phrasing)
- Each participant record has an individual "time multiplier" that acts on this. I'm sympathetic to adding comments here to clarify this.